### PR TITLE
Anwar object store

### PR DESCRIPTION
**Pull request by anwarmontasir**
Create a library the provides an "object store" to manage objects. 
This took approximately four hours.

### CODE REVIEW

**Comment by nsmeds** [comment]
Travis is failing - looks like it has to do with the `scripts` section of your package.json.

**Comment by anwarmontasir** [comment]
Yes, there was indeed an issue in package.json. It's passing now.

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (nsmeds) [inline .vscode/launch.json]: It's a good idea to add `.vscode` to your .gitignore file.

File: lib/store.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    return list.slice();
+}
+
+function remove(id) {
+    const index = list.findIndex(obj => obj._id === id);
+    if (index < 0) return { removed: false };
+    list.splice(index, 1);
+    return { removed: true };
+}
+
```
Comment (nsmeds) [inline lib/store.js]: Looks good - nice concise functions and good use of ES6.

**Pull request merged**